plugins/modules/katello_activation_key.py

```diff
@@ -251,6 +251,10 @@ def main():
             current_subscription_ids = set(item['id'] for item in current_subscriptions)
 
             if desired_subscription_ids != current_subscription_ids:
+                module.record_before('activation_keys/subscriptions', {'id': activation_key['id'], 'subscriptions': current_subscription_ids})
+                module.record_after('activation_keys/subscriptions', {'id': activation_key['id'], 'subscriptions': desired_subscription_ids})
+                module.record_after_full('activation_keys/subscriptions', {'id': activation_key['id'], 'subscriptions': desired_subscription_ids})
+
```

Comment:
I would have expected to see the 'after' parts after the actions, but i guess we can just assume, that it worked properly if we don't get an error.

Reply:
yeah, that was the assumption I've been working under. can move if you prefer.